quickref/qsharp-quick-reference.tex

```diff
@@ -198,15 +211,22 @@
     Return a value     & \texttt{return \emph{value}} \\
     Stop with an error & \texttt{fail "\emph{Error message}"} \\
   \end{keysref}
-
+   
+   \begin{keysref}{Conjugations}
+	ApplyWith & \texttt{operation \emph{Name}(\emph{in0} : \emph{type0}, ...) : \emph{returnType} \{} \newline
+                 \texttt{\hphantom{....}within \{ ... \}} \newline
+                 \texttt{\hphantom{....}apply \{ ... \}} \newline
+                 \texttt{\}} \\ 
+   \end{keysref}
+  
   \begin{keysref}{Debugging}
     Print a string     & \texttt{Message("Hello Quantum!")} \\
     Print an \newline interpolated string   
                        & \texttt{Message(\$"Value = \{\emph{val}\}")} \\
     Assert that a qubit is in $\ket{0}$ or $\ket{1}$ state   & \texttt{AssertQubit(Zero, \emph{oneQubit})}\\
     Print amplitudes \newline of wave function & \texttt{DumpMachine("dump.txt")} \\
   \end{keysref}
-    
+  
   \begin{keysref}{Qubit Allocation}
```

Comment:
Let's add syntax for allocating a mix of qubits and qubit registers to this section, since it's a very convenient shorthand.

Reply:
Could you clarify this?

Reply:
Something like `using (x, y) = (Qubit[N], Qubit())` to show that you can allocate multiple variables in one `using` statement